### PR TITLE
Clarify "Go to merge base commit" menu option text

### PR DESCRIPTION
**Pull request by mdonatas**
Based on follow-up discussion in #5538 I propose to change menu `Go to merge base commit` to **`Go to common ancestor (merge base)`**.
If not for fear of PR rejection, I'd loose the `(merge base)` part altogether and only mention `merge base` in the tooltip. Please consider this too. It would actually be closer to how we name menus/buttons today.

Completely agree with @drewnoakes 
> The trend among git UIs is towards making things easier and friendlier. We should be no different.

We also have quite a few labels which are friendly and clear and don't confuse neither novices nor pros.
`Undo last commit` instead of `Reset HEAD~1`
`Stage` instead of `Add`
`Unstage` instead of `Reset`
and so on..

### CODE REVIEW

File: GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs [hunk 1]
```diff
@@ -110,8 +110,8 @@ private IReadOnlyList<MenuCommand> CreateNavigateMenuCommands()
                 new MenuCommand
                 {
                     Name = "GotoMergeBaseCommit",
-                    Text = "Go to merge base commit",
-                    ToolTipText = "Go to the merge base commit (last common commit) between the currently checked out commit (HEAD) and the currently selected commit",
+                    Text = "Go to common ancestor (merge base)",
```
Comment (gerhardol) [inline GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs]: The Git merge-base documentation names this "Best common ancestor".
`Text = "Go to common ancestor",
ToolTipText = "Go to the best common ancestor (merge base) commit (last common commit) between the currently checked out commit (HEAD) and the currently selected commit",`

Comment (RussKie) [reply GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs]: We have too many parenthesis in the tooltip, suggest rewording to:
```
"Go to the common ancestor (merge base) commit, which is a last common commit between the currently checked out commit (HEAD) and the currently selected commit"
```

Comment (RussKie) [reply GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs]: We have too many parenthesis in the tooltip, suggest rewording to:
```
"Go to the common ancestor (merge base) commit, which is a best common commit between the currently checked out commit (HEAD) and the currently selected commit"
```

Comment (gerhardol) [reply GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs]: ok for me

Comment (drewnoakes) [reply GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs]: Agree that @RussKie's text is easier to read. Otherwise I think this PR is great 👍

Comment (mdonatas) [reply GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs]: Changed the tooltip text to suggested version.

Comment (RussKie) [reply GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs]: I think I made a typo and it should be "._..which is **the** best common commit..._" 😕

Comment (mdonatas) [reply GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs]: No problem, fixed.
